packages/paste-core/components/code-block/stories/index.stories.tsx

```diff
@@ -125,6 +125,22 @@ export const CodeBlockTabs: StoryFn = () => (
     <CodeBlockTabList>
       <CodeBlockTab>Ruby</CodeBlockTab>
       <CodeBlockTab>PHP</CodeBlockTab>
+      <CodeBlockTab>Python</CodeBlockTab>
```

Comment:
suggestion(non-blocking): if you have time it might be nice to throw this story in a fixed-width box so we can vrt the scroll

Reply:
same with the in page nav overflow example

Reply:
Great suggestion! I'll get that in